site-src/guides/tls.md

```diff
@@ -98,6 +101,56 @@ would be invalid.
 {% include 'standard/tls-cert-cross-namespace.yaml' %}
 ```
 
+## Upstream TLS
+
+Upstream TLS settings are configured using the experimental `BackendTLSPolicy`
+applied to a `Service` that accesses the backend to be targeted.
+This type of TLS is used when the backend pod has its own certificate and the
+application developer needs to convey how to successfully connect to the
+backend pod from the Gateway.
+
+### TargetRefs and TLS
+
+The `TargetRef` identifies the `Service` for which your HTTPRoute requires TLS.
+The `TLS` configuration for upstream TLS contains a required `Hostname`, and either
+`CACertRefs` or `WellKnownCACerts`.
+`Hostname` refers to the SNI the Gateway should use to connect to the backend, and
+must match the certificate served by the backend pod.
+`CACertRefs` refer to one or more PEM-encoded TLS certificates.
+If there are no specific certificates to use, then you must set `WellKnownCACerts` to
+"System" to tell the Gateway to use a set of trusted CA Certificates.  Refer to your
+implementation documentation for more details.
+
+!!! info "Restrictions"
+
+    - Cross-namespace certificate references are not allowed.
+    - Wildcard hostnames are not allowed.
+
+### Examples
+
+#### Using System Certificates
+
+In this example, the `BackendTLSPolicy` is configured to use system certificates
+to connect with a TLS-encrypted upstream connection to the pod called `dev-pod`,
+with hostname `dev-pod.example.com`, and
+served by the service called `dev-service`.
```

Comment:
Similar to above, would avoid suggesting that this refers to a single Service.
```suggestion
to connect with a TLS-encrypted upstream connection where Pods backing the
`dev` Service are expected to serve a valid certificate for `dev.example.com`.
```

Reply:
I think you meant "to avoid suggesting that this refers to a single Pod"?  Agreed.